.github/workflows/angular.yml

```diff
@@ -0,0 +1,30 @@
+name: Angular CI
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - run: npm install
+      - run: npm run build --if-present
+      - run: npm run test -- --watch=false --browsers=ChromeHeadless
+      - run: npm run e2e -- --watch=false --browsers=ChromeHeadless
```

Comment:
@YasunoriMATSUOKA 

CI failed here. (`ng e2e --protractor-config=e2e/protractor-ci.conf.js`)
Options are wrong.
Refer to the following link to fix it.

https://angular.jp/guide/testing#chrome%E3%81%A7%E3%81%AEci%E3%83%86%E3%82%B9%E3%83%88%E7%94%A8%E3%81%ABcli%E3%82%92%E8%A8%AD%E5%AE%9A%E3%81%99%E3%82%8B

Reply:
@YasunoriMATSUOKA 

CI worked!
Thank you!